src/Media_Command.php

```diff
@@ -372,10 +372,6 @@ private function needs_regeneration( $att_id, $fullsizepath ) {
 			return true;
 		}
 
-		if ( array_diff( get_intermediate_image_sizes(), array_keys( $metadata['sizes'] ) ) ) {
```

Comment:
I don't think this is the fix, because it fails the build.

Did you try including `thumbnail` as a size? See https://github.com/wp-cli/wp-cli/issues/3859#issuecomment-288072511